core/src/main/kotlin/org/lflang/generator/ts/TSGenerator.kt

```diff
@@ -451,7 +451,11 @@ class TSGenerator(
             context.unsuccessfulFinish()
         } else {
             context.finish(GeneratorResult.Status.COMPILED, codeMaps)
-            val shScriptPath = fileConfig.binPath.resolve(fileConfig.name)
+            val shScriptPath = if (GeneratorUtils.isHostWindows()) {
```

Comment:
Why replicate the logic of `getExecutableExtension` here? Seems like you can just call `fileConfig.getExecutable()`.

Reply:
You're right! I wrote it this way to see if it would work and modified the `getExecutableExtension` later. I pushed a new commit to simplify this code.